docs/transaction.md

```diff
@@ -157,70 +157,6 @@ refer to the [official documentation][transaction-iso] for more information.
 
 ## Nested Transactions
 
-Nested transactions happen when you put one `with-tx` inside another, for
```

Comment:
sorry I removed this passage as it seems to be outdated: pg2 works fine with nested `with-tx` now